static/bidder-params/rubicon.json

```diff
@@ -38,6 +38,10 @@
         "type": "array"
       }
     },
+    "bidonmultiformat": {
```

Comment:
@SerhiiNahornyi This PR LGTM. Can you please open up a docs PR for this new parameter if you already haven't done so? Thanks!

Reply:
We don't really want general pubs to use this param yet, but I added a note about it as a "beta" parameter in https://github.com/prebid/prebid.github.io/pull/4212